san/v2_metrics_list.py

```diff
@@ -0,0 +1,63 @@
+V2_METRIC_QUERIES = [
+    'daily_active_addresses',
+    'daily_avg_marketcap_usd',
+    'daily_closing_marketcap_usd',
+    'daily_stack_circulation',
+    'daily_trading_volume_usd',
+    'mean_realized_price_usd_10y',
+    'mean_realized_price_usd_180d',
+    'mean_realized_price_usd_1d',
+    'mean_realized_price_usd_20y',
+    'mean_realized_price_usd_2y',
+    'mean_realized_price_usd_30d',
+    'mean_realized_price_usd_365d',
+    'mean_realized_price_usd_3y',
+    'mean_realized_price_usd_5y',
+    'mean_realized_price_usd_60d',
+    'mean_realized_price_usd_7d',
+    'mean_realized_price_usd_90d',
+    'mvrv_usd_10y',
+    'mvrv_usd_180d',
+    'mvrv_usd_1d',
+    'mvrv_usd_20y',
+    'mvrv_usd_2y',
+    'mvrv_usd_30d',
+    'mvrv_usd_365d',
+    'mvrv_usd_3y',
+    'mvrv_usd_5y',
+    'mvrv_usd_60d',
+    'mvrv_usd_7d',
+    'mvrv_usd_90d',
+    'stack_age_consumed',
+    'stack_circulation_10y',
+    'stack_circulation_180d',
+    'stack_circulation_1d',
+    'stack_circulation_20y',
+    'stack_circulation_2y',
+    'stack_circulation_30d',
+    'stack_circulation_365d',
+    'stack_circulation_3y',
+    'stack_circulation_5y',
+    'stack_circulation_60d',
+    'stack_circulation_7d',
+    'stack_circulation_90d',
+    'stack_cumulative_age_consumed',
+    'stack_liveliness',
+    'stack_mean_age_days',
+    'stack_realized_cap_usd',
+    'stack_realized_cap_usd_10y',
+    'stack_realized_cap_usd_180d',
+    'stack_realized_cap_usd_1d',
+    'stack_realized_cap_usd_20y',
+    'stack_realized_cap_usd_2y',
+    'stack_realized_cap_usd_30d',
+    'stack_realized_cap_usd_365d',
+    'stack_realized_cap_usd_3y',
+    'stack_realized_cap_usd_5y',
+    'stack_realized_cap_usd_60d',
+    'stack_realized_cap_usd_7d',
+    'stack_realized_cap_usd_90d',
+    'stack_total_age',
+    'token_velocity',
+    'transaction_volume'
+]
```

Comment:
This list is outdated, this is the current list: https://api.santiment.net/graphiql?variables=&query=%7B%0A%20%20getAvailableMetrics%0A%7D%0A

Reply:
It should be okay now